CMakeLists.txt

```diff
@@ -167,17 +167,17 @@ endif()
 #  so we default to yes on iOS.
 #  TODO: Windows can support libusb, the hid.c file just depends on Unix APIs
 if((WINDOWS AND NOT WINDOWS_STORE) OR IOS OR TVOS OR ANDROID)
```

Comment:
Is `NOT WINDOWS_STORE` here is a product of copy & paste or do we really support UWP with libusb?

Reply:
I'm not sure if libusb works on UWP, but our libusb support does work on Windows. I tested it this week.

Reply:
This part of the cmake'ry needs revision, then. As for windows, is the native driver not good enough?

Reply:
The Windows libusb is definitely quirky and if we change this, we shouldn't enable it by default. For example, sending some reports to a DualSense controller will hang for a full second - and it doesn't do that on any other hidapi backend.